packages/calling-component-bindings/src/utils/callUtils.ts

```diff
@@ -146,8 +148,8 @@ export const _convertParticipantState = (participant: RemoteParticipantState): P
   /* @conditional-compile-remove(remote-ufd) */
   if (
     participant.diagnostics &&
-    participant.diagnostics['ServerConnection'] &&
-    participant.diagnostics['ServerConnection']?.value === false
+    participant.diagnostics['ServerConnection' as ServerDiagnosticType] &&
```

Comment:
why is `as` needed here?

Reply:
Looks like it should be `serverConnection` instead of `ServerConnection`?

![image](https://github.com/user-attachments/assets/8dd642d6-574e-4d05-9ff2-aa87259d75c1)

Reply:
Hmm, that is interesting! Since "ServerConnection" is the value coming from sdk! I guess I can do .toLowerCase() to make sure it matches anyway, but would be a bug for calling sdk

Reply:
Oh! Yes get them to fix that, I would keep the `as` then and if we remember when they fix it on their end we can remove